tests/skr/proxy.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
is the purpose of this container that grpcurl isn't on the test runner machine? It seems like indirection to wrap the grpc server with an http server if we could call the grpc directly from test.py

Reply:
It's mostly to test/demonstrate them as sidecars to some primary container since users would more likely use that pattern